### ISSUE TITLE
Fine-tunning Lag-Llma on dataset with different frequencies

### ISSUE THREAD

**Issue by JuanDiegoArango** [time-series-foundation-models/lag-llama#91]
Hi, I am working on an experiment, and I have the following query:

I want to perform a fine-tunning over Lag-Llama using a set of series time with different frequencies.

Does anyone know if it Can be done?

Thanks.

**Comment by ashok-arjun** [comment]
Yes. It can be done. Lag-Llama does not take the frequency as input; it only sees the time series.
Finetuning jointly on all these time series with different frequencies may/may not work; only finetuning will tell you.
